frontend/src/common/components/dashboard/components/banner/Banner.scss

```diff
@@ -0,0 +1,76 @@
+@import "../../../../../themes/orbcStyles";
+
+.layout-banner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 2.5rem;
+  padding-bottom: 2.5rem;
+  position: relative;
+  min-height: 3rem;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.12);
+
+  & &__left {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  &__text-section {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  & &__suspended {
```

Comment:
I thought that the suspended warning message wasn't a snackbar message, but instead a part of the banner at the top. I'll check with Girish. In any case, I made css style changes to refactor the banner and tabbanner styles, and I'll remove the &__suspended styles in case Girish decides that it should in fact be used as snackbar instead.

Reply:
Ok he said it should be a snackbar. I'll make changes for this.

Reply:
The suspend snackbar will stay in that top-center location indefinitely (as long as the company is suspended), and despite it being a snackbar it doesn't work the same as the other snackbars (eg. position of display, not using the SnackBarContext since its display only depends on 'isCompanySuspended' and there's no need to call 'setShowSnackbar')